### PR TITLE
Move usage of `native-keymap` to the main process

### PR DESCRIPTION
**Pull request by alexdima**

### CODE REVIEW

File: src/vs/code/electron-main/app.ts [hunk 1]
```diff
@@ -82,6 +82,7 @@ import { generateUuid } from 'vs/base/common/uuid';
 import { VSBuffer } from 'vs/base/common/buffer';
 import { EncryptionMainService, IEncryptionMainService } from 'vs/platform/encryption/electron-main/encryptionMainService';
 import { ActiveWindowManager } from 'vs/platform/windows/common/windowTracker';
+import { IMainKeyboardLayoutService, MainKeyboardLayoutService } from 'vs/platform/keyboardLayout/electron-main/keyboardLayoutMainService';
```
Comment (bpasero) [inline src/vs/code/electron-main/app.ts:85]: Just a detail, but most of our `main` services are called `I<Kind>MainService`, so I suggest to call it `IKeyboardLayoutMainService`

File: src/vs/code/electron-main/app.ts [hunk 2]
```diff
@@ -531,6 +522,10 @@ export class CodeApplication extends Disposable {
 		const encryptionChannel = createChannelReceiver(encryptionMainService);
 		electronIpcServer.registerChannel('encryption', encryptionChannel);
 
+		const mainKeyboardLayoutService = accessor.get(IMainKeyboardLayoutService);
+		const mainKeyboardLayoutChannel = createChannelReceiver(mainKeyboardLayoutService);
+		electronIpcServer.registerChannel('mainKeyboardLayout', mainKeyboardLayoutChannel);
```
Comment (bpasero) [inline src/vs/code/electron-main/app.ts:527]: Again just a small detail, typically our channel names don't reference the main side, so I suggest `keyboardLayout` as channel name.

**Review by bpasero — APPROVED** [review]
Very cool, just a few comments around naming, overall looks great 👍

File: src/vs/code/electron-main/app.ts (hunk 2 above)
Comment (alexdima) [reply src/vs/code/electron-main/app.ts]: :+1:

File: src/vs/code/electron-main/app.ts (hunk 1 above)
Comment (alexdima) [reply src/vs/code/electron-main/app.ts]: :+1:

**Pull request merged**